branchpro/posterior.py

```diff
@@ -256,6 +256,105 @@ def get_intervals(self, central_prob):
 
         return intervals_df
 
+    def proportion_time_r_more_than_1(self, central_prob=.95):
```

Comment:
We could add another optional argument to both of these methods allowing the user to choose either posterior mean (as it is now) or median for the central calculation